CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [v4.4.4](https://github.com/codeigniter4/CodeIgniter4/tree/v4.4.4) (Unreleased)
+[Full Changelog](https://github.com/codeigniter4/CodeIgniter4/compare/v4.4.3...v4.4.4)
+
+### Fixed Bugs
+
+* fix: [CURLRequest] skip hostname checks if options 'verify' false by @NicolaeIotu in https://github.com/codeigniter4/CodeIgniter4/pull/8258
+
```

Comment:
I think that this file is handled automatically when we prepare the release. @kenjis can confirm or deny.

Reply:
Yes, it is generated by GitHub.
Please add in changelog in user guide.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/user_guide_src/source/changelogs/v4.4.4.rst#breaking

Reply:
Thanks!